cp3pt0-deployment/setup_singleton.sh

```diff
@@ -227,4 +235,16 @@ function get_and_validate_arguments() {
     get_control_namespace
 }
 
+function accept_license() {
+    if [[ $LICENSE_ACCEPT == "true" ]]; then
+        certmanager_cr=$(${OC} get certmanager --no-headers | awk '{print $1}')
```

Comment:
I think it would be new API for cert-manager
```
oc get certmanagerconfig.operator.ibm.com
```

Reply:
ok I'll update that